bin/keystone-dev.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * This file is based on @keystonejs/keystone/bin/commands/dev.js
+ * The main reason is to add PORT environment support for `yarn dev` command!
+ * You can now use command like so: `PORT=3002 yarn dev`
+ * Or put PORT inside your .env file
+ */
+const path = require('path')
+
+const { getEntryFileFullPath, executeDefaultServer } = require('@keystonejs/keystone/bin/utils')
```

Comment:
I guess this deps should be in root package.json...Seems like there's "hidden" usage of node_modules, which can differs across all apps...

Reply:
I'm not sure. You will use it from different apps and each app can have their own version.